wiki/Modding/en.md

```diff
@@ -1,98 +1,237 @@
+The actual song setup settings does not impact game play, but it is important to give correct names.
+The artist name, title of the song, source (if applicable), and tags should all be the same in each difficulty.
 
-What to avoid in your mod post
-------------------------------
+Audio lead-in times are usually the same, but can differ depending on when the first note is played.
+The `Enable Countdown` checkbox should be either enabled or disabled for all difficulties.
 
-1.  **Vague Terms:**
-    -   **[Spacing](Spacing "wikilink")** When offering help to inexperienced mappers, you should explain **why** it is incorrect spacing. This will help the new mapper grasp a better understanding of beatmapping. On the other hand, more experienced mappers rarely make accidental spacing mistakes, so please explain **why** you think the spacing should be changed.
-    -   **[New Combo/Remove Combo](Combos "wikilink")** Many times adding new combos will help a map. However, because the modder does not explain **why**, the mapper may not understand the reason behind doing so and will not change the combo. It is always a good idea to support your suggestion with **why** it is important.
-    -   **1 grid to the right** It is not always obvious **why** a hit object should be moved a grid up/right/down/left. It is fine to suggest this, but explain **why** it should be moved (typically it's to make a pattern symmetrical). This sort of suggestion, however, does not offer much to improve a beatmap. Keep in mind a mod post should not only consist of such suggestions.
-    -   **This shape is ugly** Whether the shape is made of [hit circles](Hit_circle "wikilink"), [sliders](sliders "wikilink") or both, the mapper obviously will not understand what you are asking for. What looks aesthetically pleasing to the mapper may not always look aesthetically pleasing to you. Please understand that the osu! community comes from across the world and respect each player's opinions. Instead of stating the slider is outright ugly, it is better to say "This slider looks weird in my opinion" and then to offer an alternative (via an in-post attachment or an external link).
+#### Difficulty
 
-2.  **Offensive Language** Even if the mapper is a close friend, please try to keep the mod post clean. It is understandable that certain patterns may seem completely illogical to you, but that is not a reason for insulting the mapper. Refraining from terms such as "WTF is this?" or "This is completely stupid" can help to repress any potential argument and create a better osu! environment.
+The difficulty settings can be found in the song setup dialog.
+These are important to game play so that the song is at a fair level of difficulty for the varying difficulties a mapset can have.
+
```

Comment:
just a quick note that you could refer to https://osu.ppy.sh/wiki/Standard_Ranking_Criteria#Difficulty-specific regarding composition of difficulties